src/core_plugins/input_control_vis/public/components/editor/control_editor.less

```diff
@@ -0,0 +1,4 @@
+.controlEditorPanel {
+  z-index: 1;
```

Comment:
Why are we setting this z-index? We're also setting it here:
`<EuiPanel grow={false} className="controlEditorPanel" style={{ zIndex: 1 }}>`

Reply:
I will remove the style block from the html - that one is extra.

The z-index in the css is needed otherwise the select menu for index pattern or field is behind the "control types" select